webpack.config.js

```diff
@@ -0,0 +1,73 @@
+var CleanWebpackPlugin = require('clean-webpack-plugin');
+var Path               = require('path');
+var Config             = require('webpack-config').default;
+var ExtractTextPlugin  = require('extract-text-webpack-plugin');
+var Webpack            = require('webpack');
+
+module.exports = new Config().merge({
+    context: Path.join(__dirname, './Resources/public'),
+    devtool: "source-map",
+    entry: {
+        app: [
+            './javascript/app.js', './scss/theme.scss'
+        ]
+    },
+    output: {
+        path: Path.join(__dirname, './Resources/public/build'),
+        publicPath: '/build/',
+        filename: '[name].js'
+    },
+    resolve: {
+        modules: [
+            Path.resolve('./node_modules')
+        ]
+    },
+    module: {
+        rules: [
+            {
+                test: /\.(jpe?g|png|gif|svg|woff|woff2|ttf|otf|eot)$/i,
+                loader: 'file-loader?hash=sha512&digest=hex&name=[name].[ext]'
+            },
+            {
+                test: /.js$/,
+                loader: 'babel-loader?presets[]=es2015-ie',
+                exclude: /node_modules|@wearejust/
+            },
+            {
+                test: /\.scss$/,
+                loaders: ExtractTextPlugin.extract({
+                    fallback: 'style-loader',
+                    use: [
+                        { loader: "css-loader", options: { sourceMap: true } },
+                        {
+                            loader: 'postcss-loader', options: {
+                            sourceMap: true,
+                            ctx: {
+                                autoprefixer: {
+                                    browsers: ['last 4 versions']
+                                }
+                            }
+                        }
+                        },
+                        { loader: "sass-loader", options: { sourceMap: true } }
+                    ],
+                }),
+            },
+        ]
+    },
+    plugins: [
+        new CleanWebpackPlugin(['public/build/*']),
+        new Webpack.LoaderOptionsPlugin({
+            minimize: true,
+            debug: false,
+        }),
+        new Webpack.optimize.UglifyJsPlugin({
+            compress: true,
+            sourceMap: false
+        }),
```

Comment:
We've noticed that using the minimize and compress here results in differences in results sometimes. Rather remove lines 60-67.

Reply:
Also from devDependencies? I quess ```        "@wearejust/trace": "^2.0.5",
``` must be in there right?

Reply:
@Emroni No minimize at all?